### PR TITLE
[JBPM-9474] Produce events to Kafka by Signal or Messages Events

### PR DESCRIPTION
**Pull request by fjtirado**
Adding listener for signal and messages

**JIRA**: 

[JBPM-9474](https://issues.redhat.com/browse/JBPM-9474)

**referenced Pull Requests**: 

https://github.com/kiegroup/jbpm/pull/1808
https://github.com/kiegroup/drools/pull/3264
https://github.com/kiegroup/droolsjbpm-integration/pull/2322

### CODE REVIEW

**Review by afalhambra — CHANGES_REQUESTED** [review]
Minor typos. Otherwise looks good to me.

**Review by gmunozfe — APPROVED** [review]
Looks good to me, good job @fjtirado !

**Review by afalhambra — APPROVED** [review]
Good job @fjtirado! thanks

**Comment by fjtirado** [comment]
jenking run cdb

**Comment by fjtirado** [comment]
jenkins run fdb

File: kie-api/src/main/java/org/kie/api/event/process/SignalEvent.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kie.api.event.process;
+
+/**
+ * An event when a signal is thrown
+ */
+public interface SignalEvent extends ProcessNodeEvent {
+    /**
```
Comment (elguardian) [inline kie-api/src/main/java/org/kie/api/event/process/SignalEvent.java:22]: add a string for implementation

**Review by elguardian — CHANGES_REQUESTED** [review]
changes required.

File: kie-api/src/main/java/org/kie/api/event/process/SignalEvent.java (hunk 1 above)
Comment (fjtirado) [reply kie-api/src/main/java/org/kie/api/event/process/SignalEvent.java:22]: not doing it, currently skipping information from process, to be done with another JIRA

**Review by elguardian — APPROVED** [review]

**Pull request merged**